Python/pywarpx/fields.py

```diff
@@ -140,6 +140,31 @@ def __getitem__(self, index):
         elif self.dim == 3:
             return self._getitem3d(index)
 
+    def _find_start_stop(self, ii, nn, ngrow, d):
+        """Given the input index, calculate the start and stop range of the indices.
+        - ii: index, either a slice object or an integer
+        - nn: the global number of cells along the specified direction
```

Comment:
Just one param skipped :)
```suggestion
        - nn: the global number of cells along the specified direction
        - ngrow: ...
```

Reply:
That's what I get for hurrying :)